src/libexpr/eval.cc

```diff
@@ -1094,13 +1095,29 @@ void EvalState::callPrimOp(Value & fun, Value & arg, Value & v, const Pos & pos)
     }
 }
 
+void fnStart(const Pos & pos) {
+    auto now = std::chrono::high_resolution_clock::now();
+    auto duration = now.time_since_epoch();
+    auto ns = std::chrono::duration_cast<std::chrono::nanoseconds>(duration);
```

Comment:
This should all be wrapped in `if (verbosity >= lvlDebug) { ... }`.

Reply:
Without this change, it'll be crazy expensive. I was hoping you'd be able to point me to exactly this! :)